contributor-guides/submitting-a-recipe.md

```diff
@@ -44,7 +44,7 @@ Examples that would not be considered recipes:
 - A walk-through that teaches more than one task. (e.g. "How to set up Vue, TypeScript, and Partytown in Astro")
 - A conceptual explanation of a topic, instead of a how-to guide that leads the reader through completing a specific task. (e.g. "How partial hydration works in Astro")
 
-Like all contributions to our docs, take a look at the [WRITING.md](https://github.com/withastro/docs/blob/main/WRITING.md
+Like all contributions to our docs, take a look at the [WRITING.md](https://github.com/withastro/docs/blob/main/contributor-guides/writing-and-style-guide.md
```

Comment:
I would suggest updating the `WRITING.md` here as well, since we've also done it anywhere else in this file.

```suggestion
Like all contributions to our docs, take a look at the [Writing Guide](https://github.com/withastro/docs/blob/main/contributor-guides/writing-and-style-guide.md
```